### PR TITLE
Feat/grants 2479

### PR DESCRIPTION
**Pull request by salfedev**

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -2324,6 +2324,93 @@ const questionBank = {
           },
           yarKey: "separatorType",
         },
+        {
+          key: 'gantry',
+          order: 142,
+          title: 'Do you want to add a gantry?',
+          hint: {
```
Comment (zavies97) [inline app/config/question-bank.js:2331]: hard coded text so cant approve. This needs to come from the database

**Review by zavies97 — DISMISSED** [review]

File: app/config/question-bank.js (hunk 1 above)
Comment (salfedev) [reply app/config/question-bank.js:2331]: yup you're right

File: app/config/question-bank.js [hunk 2]
```diff
@@ -2266,12 +2266,12 @@ const questionBank = {
           hint: {
             text: 'Capable to processing at least 3m3 of slurry per hour and producing at least 25% dry matter. '
           },
-          url: 'separator-type',
-          baseUrl: 'separator-type',
-          backUrl: 'separator',
-          nextUrl: 'other-items',
-          preValidationKeys: ['separator'],
-          type: 'single-answer',
+          url: "separator-type",
```
Comment (zavies97) [inline app/config/question-bank.js:2269]: dont know why your PRs keep doing double quotes xD

Comment (salfedev) [reply app/config/question-bank.js:2269]: eslint said so

**Review by zavies97 — APPROVED** [review]

**Pull request merged**